docs/user-guide/getting-started/installation.md

```diff
@@ -11,15 +11,6 @@
 pip install "kaiko-eva[vision] @ git+https://github.com/kaiko-ai/eva.git"
 ```
 
-- To be able to use the existing configs, download them into directory where you installed *eva*. You can get them from our blob storage with:
```

Comment:
can you also change line 11 to `pip install "kaiko-eva[vision]@ git+https://github.com/kaiko-ai/eva.git"`?